### PR TITLE
Add example circle_natural_disasters.

### PR DESCRIPTION
**Pull request by chanwutk**
Please:
- [ ] Make your pull request atomic, fixing one issue at a time unless there are many relevant issues that cannot be decoupled.
- [ ] Provide a test case & update the documentation under `site/docs/`
- [x] Make lint and test pass. (Run `npm run lint`, `npm run test`, `npm run build:examples`, `npm run test:runtime:generate`.)
- [ ] Make sure you have rebased onto the `master` branch.
- [ ] Provide a concise title so that we can just copy it to our release note.
  - Use imperative mood and present tense.
  - Mention relevant issues. (e.g., `#1`)
  
Pro-Tip: https://medium.com/@greenberg/writing-pull-requests-your-coworkers-might-enjoy-reading-9d0307e93da3 is a nice article about writing a nice pull request.

This is for issue #3232

### CODE REVIEW

**Comment by domoritz** [comment]
Why did you close it?

**Comment by chanwutk** [comment]
I was confusing with the other branch I PR to the vega-dataset that you told me to increase the version number, so I thought I should close this one and do it again.

**Comment by domoritz** [comment]
There is no need to close a PR even if you make major changes. Just push them to the branch to update the PR.

File: _data/versions.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-vega: 3.0.8
```
Comment (domoritz) [inline _data/versions.yml]: Please remove this

File: _includes/docs_toc.md [hunk 2]
```diff
@@ -36,15 +36,16 @@
         - [Example]({{site.baseurl}}/docs/calculate.html#example)
     - [Filter]({{site.baseurl}}/docs/filter.html)
         - [Filter Expression]({{site.baseurl}}/docs/filter.html#expression)
-        - [Filter Object]({{site.baseurl}}/docs/filter.html#filter-object)
```
Comment (domoritz) [inline _includes/docs_toc.md]: Don't change the toc in this pr

File: _data/examples.json [hunk 3]
```diff
@@ -87,7 +87,7 @@
     },
     {
       "name": "circle_natural_disasters",
-      "description": "Visualization of global deaths from natural disasters.",
+      "description": "Visualization of global deaths from natural disasters. Copy of chart from [Our World in Data](https://ourworldindata.org/natural-catastrophes)",
```
Comment (kanitw) [inline _data/examples.json]: "This is a copy of..." ?

**Pull request merged**